sdk/storage/microsoft-azure-storage-blob/pom.xml

```diff
@@ -76,7 +76,7 @@
         <dependency>
             <groupId>org.spockframework</groupId>
             <artifactId>spock-core</artifactId>
-            <version>1.3-groovy-2.5</version> <!-- {x-version-update;org.spockframework:spock-core;external_dependency} -->
+            <version>2.0-M4-groovy-2.5</version> <!-- {x-version-update;org.spockframework:spock-core;external_dependency} -->
```

Comment:
Let's make an exception for Track 1 Blobs to continue using the old version of Spock